### PR TITLE
Partial Custom Tabs update

### PR DESCRIPTION
**Pull request by IanStanion-google**
Added new content related to new Partial Custom Tabs documentation

Changes proposed in this pull request:

- Implemented proposed changes to the Overview page for Custom Tabs
- Implemented proposed changes to the Integration Guide page for Custom Tabs
- Implemented proposed changes to the Headers page for Custom Tabs

### CODE REVIEW

**Review by ali-spivak — APPROVED** [review]

**Review by ali-spivak — APPROVED** [review]

File: site/en/docs/android/custom-tabs/integration-guide/index.md [hunk 1]
```diff
@@ -44,6 +44,35 @@ CustomTabsIntent customTabsIntent = builder.build();
...
+
+The snippet below changes the launch height of the Custom Tab. CustomTabsIntent.Builder has a new method `setInitialActivityHeightPx(int initialHeightPx)` that allows you to define the initial launch height in pixels. As a prerequisite to using Partial Custom Tabs, you will need to [connect to the Custom Tabs Service](/docs/android/custom-tabs/integration-guide/#connect-to-the-custom-tabs-service) or launch the custom tabs intent using [Activity#startActivityForResult()](/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)).
+
+There are API's that do not work when building an Intent for Partial Custom Tabs:
+
+-   CustomTabColorScheme.navigationBarColor
+-   CustomTabColorScheme.navigationBarDividerColor
+
+Partial Custom Tabs will inherit the host app's color scheme for the UI components above. You will be responsible for ensuring visual consistency for these components before launching a Partial Custom Tab.
```
Comment (jpmedley) [inline site/en/docs/android/custom-tabs/integration-guide/index.md:62]: >UI components above

Which UI components are you talking about? The last thing I saw was a couple of properties. Please clarify this.

File: site/en/docs/android/custom-tabs/integration-guide/index.md [hunk 2]
```diff
@@ -44,6 +44,35 @@ CustomTabsIntent customTabsIntent = builder.build();
...
+
+These new APIs are available from Chrome M107 and onwards.
```
Comment (jpmedley) [inline site/en/docs/android/custom-tabs/integration-guide/index.md:64]:
```suggestion
These new APIs are available from Chrome 107 and onwards.
```

File: site/en/docs/android/custom-tabs/integration-guide/index.md [hunk 3]
```diff
@@ -251,6 +280,21 @@ public static final int TAB_SHOWN = 5;
 public static final int TAB_HIDDEN = 6;
 ```
 
+```java
+/**
+* Called when Custom Tab is resized in its height. This is applicable when users resize a
```
Comment (jpmedley) [inline site/en/docs/android/custom-tabs/integration-guide/index.md:285]:
```suggestion
* Called when Custom Tab's height is resized. This applies when users resize a 
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
This is a nice bit of writing. Most of my comments relate to norms and best practices of writing and technical writer that we need to start following. 

Let me know if you have any questions.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**